### PR TITLE
docs: add docs about the admin panel

### PR DESCRIPTION
**Pull request by leafty**

### CODE REVIEW

File: docs/how-to-guides/admin/crc.rst [hunk 1]
```diff
@@ -1,8 +1,54 @@
...
+.. image:: ../../_static/images/admin-panel.png
+    :width: 100%
+    :align: center
+    :alt: Screenshot of the admin panel
+
+The admin panel lets you configure resource pools and classes to be used in
+interactive sessions. You can also assign users to private resource pools to
+give them access to specific compute resources.
+
 Working with the CRC service
```
Comment (lokijuhy) [inline docs/how-to-guides/admin/crc.rst:46]:
```suggestion
Working with the Compute Resource service
```
I recognize that this isn't from your PR, but while we're here, could you please swap out the CRC acronyms for their full name? I wouldn't expect admins to know what CRC is. For that matter, I'm not sure what the last C stands for... Compute Resource ...?

**Review by lokijuhy — CHANGES_REQUESTED** [review]

File: docs/how-to-guides/admin/crc.rst (hunk 1 above)
Comment (rokroskar) [reply docs/how-to-guides/admin/crc.rst:46]: ... Control ;)

Comment (leafty) [reply docs/how-to-guides/admin/crc.rst]: Done in c9bf37c6.

File: docs/how-to-guides/admin/crc.rst [hunk 2]
```diff
@@ -1,8 +1,54 @@
...
+have the `renku-admin` role.
+To do so, first sign in to the Keycloak console and open the `Renku` realm.
+Then, search for your own user and head to the `Role mapping` tab.
+
+.. image:: ../../_static/images/admin-keycloak-role-mapping.png
+    :width: 100%
+    :align: center
+    :alt: Role mapping tab in Keycloak
+
+There you can check if `renku-admin` is listed as a role; if not, you can assign
```
Comment (rokroskar) [inline docs/how-to-guides/admin/crc.rst:17]:
```suggestion
There you can check if ``renku-admin`` is listed as a role; if not, you can assign
```

File: docs/how-to-guides/admin/crc.rst [hunk 3]
```diff
@@ -1,8 +1,54 @@
...
+the role by clicking `Assign Role` and selecting `renku-admin` from the list.
+Confirm the assignment by clicking `Assign`; the `renku-admin` role should now
```
Comment (rokroskar) [inline docs/how-to-guides/admin/crc.rst:19]:
```suggestion
Confirm the assignment by clicking `Assign`; the ``renku-admin`` role should now
```

File: docs/how-to-guides/admin/crc.rst [hunk 4]
```diff
@@ -1,8 +1,54 @@
...
+be listed for your user.
+
+Once this step is done, you can sign out and in again to RenkuLab to refresh
+your user permissions.
+
+Using the admin panel in RenkuLab
+---------------------------------
+
+If your user has the `renku-admin` role, you should now have access to the admin
```
Comment (rokroskar) [inline docs/how-to-guides/admin/crc.rst:28]:
```suggestion
If your user has the ``renku-admin`` role, you should now have access to the admin
```

**Review by rokroskar — COMMENTED** [review]
this is great, thanks @leafty! Just a nit-pick about using inline code styling for the role...

**Review by lokijuhy — APPROVED** [review]

**Pull request merged**